srcpkgs/sioyek/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'sioyek'
+pkgname=sioyek
+version=1.4.0
+revision=1
+build_style=qmake
+#make_build_args="pdf_viewer_build_config.pro"
+#make_install_args="sioyek pdf_viewer/prefs.config pdf_viewer/prefs_user.config
+# pdf_viewer/keys.config pdf_viewer/keys_user.config pdf_viewer/shaders
+# tutorial.pdf"
+hostmakedepends="zlib-devel qt5-qmake qt5-host-tools mupdf-devel
```

Comment:
These `-devel` packages should be in `makedepends`.

Reply:
Done.